src/components/organisms/NewProfileModal/components/header/ProfileModalBasicTextInfo/ProfileModalBasicTextInfo.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import classNames from "classnames";
+
+import { DEFAULT_PARTY_NAME } from "settings";
+
+import { User } from "types/User";
+import { ContainerClassName } from "types/utility";
+
+import { WithId } from "utils/id";
+
+import { useIsCurrentUser } from "hooks/useIsCurrentUser";
+import { useIsOnline } from "hooks/useIsOnline";
+import { useSovereignVenue } from "hooks/useSovereignVenue";
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+
+import { UserStatusDropdown } from "components/atoms/UserStatusDropdown";
+
+import "./ProfilModalBasicTextInfo.scss";
+
+export interface ProfileModalBasicTextInfoProps extends ContainerClassName {
+  user: WithId<User>;
+}
+
+export const ProfileModalBasicTextInfo: React.FC<ProfileModalBasicTextInfoProps> = ({
+  user,
+  containerClassName,
+}) => {
+  const venueId = useVenueId();
+  const { sovereignVenue } = useSovereignVenue({ venueId });
+
+  const { lastSeenIn } = useIsOnline(user.id);
+  const isCurrentUser = useIsCurrentUser(user);
+  const { user: firebaseUser } = useUser();
+
+  const lastVenue = lastSeenIn ? Object.keys(lastSeenIn)[0] : undefined;
+
+  const userStatus =
+    sovereignVenue?.showUserStatus &&
+    sovereignVenue?.userStatuses?.length &&
+    sovereignVenue?.userStatuses?.[0];
```

Comment:
```suggestion
    sovereignVenue?.showUserStatus &&
    sovereignVenue?.userStatuses?.[0];
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/53d93fd0cceb2fc3b03502c6df2e213fb9dce7be